solr/core/src/java/org/apache/solr/search/facet/FacetRangeParser.java

```diff
@@ -49,6 +49,7 @@ public FacetRange parse(Object arg) throws SyntaxError {
     facet.gap = getVal(m, "gap", required);
     facet.hardend = getBoolean(m, "hardend", facet.hardend);
     facet.mincount = getLong(m, "mincount", 0);
+    facet.dv = getBoolean(m, "dv", false);
```

Comment:
Is there a need to switch between the codepaths based on a request param? Ideally, if the field is a single valued numeric docvalue field use the new codepath, or else use the old. Is there any scenario where a user might want to use the older code path for single valued fields?